modules/ROOT/pages/deploying-to-your-own-servers.adoc

```diff
@@ -150,7 +150,8 @@ This document focuses on deploying to on-premises servers, for more information
 
 Upload a new file for deployment. Click the *Choose file* button, select your application file, and then click *Open*. You can also click the *Get from sandbox* button to copy a file from a non-production environment into your current environment. If the *Get From Sandbox* button does not appear, you may first need to create a non-production environment to view this option.
 
-[Note] The application file size limit is 200 MB.
+[NOTE] The application file size limit is 100 MB.
```

Comment:
Not quite sure. Didn't we raised the file size to 200MB @mcocaro ???

Reply:
Confirmed, we are using 200MB as default value: https://github.com/mulesoft-ops/formula-application-manager/blob/master/application-manager/files/opt/cloudhub/override.properties#L5

Reply:
You're right. I think this change got clobbered in all the branching. Fixed.